app/views/events/_form.html.erb

```diff
@@ -10,66 +10,66 @@
       </ul>
     </div>
   <% end %>
-<div class="container-fluid" style="float:center; width:70%; padding-top: 0%;">
```

Comment:
please remove inline style if possible. you did this on app/views/donations/index.html.erb line 2.